packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/services/pin_code_service.dart

```diff
@@ -18,4 +18,7 @@ abstract class PinCodeService {
 
   /// Returns the pin code or null if it is not saved in the device memory yet.
   Future<String?> getPinCode();
+
+  /// Stores the encrypted pin code in the device secure storage.
+  Future<bool> savePinCodeInSecureStorage(String pinCode);
```

Comment:
Please mention this in the changelog and update the `encryptPinCode` comment

Reply:
Mentioned in the changelog 😄